analysis_driver/quality_control/relatedness.py

```diff
@@ -52,16 +56,32 @@ def relatedness_file(self):
         return os.path.join(self.job_dir, self.dataset.name + '.relatedness2')
 
     def write_results(self, gel_lines, egc_lines):
-        header = '\t'.join(['S1', 'S1 Family ID', 'S1 Relationship', 'S2', 'S2 Family ID', 'S2 Relationship'])
+        gel_header = '\t'.join(['S1',
```

Comment:
The header order for this family check file should be S1 Family ID
```python
['Family ID', 'S1', 'S1 Relationship', 'S2', 'S2 Relationship', 'Relatedness']
```

Reply:
Will also change internal sample ID to user sample ID in output file (making a note of it so I don't forget)